src/include/duckdb/common/column_index.hpp

```diff
@@ -55,8 +77,57 @@ struct ColumnIndex {
 	vector<ColumnIndex> &GetChildIndexesMutable() {
 		return child_indexes;
 	}
+	//! Creates a copy of the ColumnIndex, up to a maximum depth, stripping all children beyond that
+	ColumnIndex CreateSubset(idx_t max_depth, bool &reached_end) const {
```

Comment:
remove, unused

Reply:
Actually.. hold off on that, because for VARIANT we *will* need this, `vector<idx_t>` won't cut it